pkg/kubernetes/views/route-panel.html

```diff
@@ -5,8 +5,8 @@
             <i class="pficon pficon-route"></i>
         </div>
         <div class="listing-actions">
-            <button title="Delete" class="btn btn-danger btn-delete pficon pficon-delete" disabled></button>
-            <button title="Adjust" class="btn btn-default pficon pficon-edit" disabled></button>
+            <button id="adjust-entity-btn" title="Adjust"  class="btn btn-default pficon pficon-edit"  data-toggle="modal" data-target="#adjust-route-dialog"  data-id="{{item.id}}" data-key="{{item.key}}" data-key="{{item.key}}"></button>
```

Comment:
Please remove id="xxx" from this line. It is invalid to use the same id in mulitple tags. This panel is displayed multiple times per page.